.github/workflows/benchmark.yml

```diff
@@ -42,3 +42,24 @@ jobs:
         with:
           command: bench
           args: --benches --features z3,boolector --locked
+
+      - name: Generate Metadata
+        run: |
+          echo "SHORT_SHA=$(echo $GITHUB_SHA | cut -c-8)" >> $GITHUB_ENV
```

Comment:
Interesting, I didn't knew, that you can pass data between steps through the environment. I think usually people do that with "outputs" tough, because it gives you the benefit, that you have to explicitly reference the step, where your are using the data from. ([https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#using-workflow-commands-to-access-toolkit-functions](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#using-workflow-commands-to-access-toolkit-functions))

Reply:
Done.